provisioning/osx/configure/src/config.spec.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import { getServices, getComposeFiles } from './config';
+
+describe('config', () => {
+  const config = {
+    stacks: [
+      {
+        name: 'services',
+        'compose-files': ['services.yml'],
+        services: [{ name: 'visualizer' }],
+      },
+      {
+        name: 'app',
+        'compose-files': ['app.yml'],
+        services: [{ name: 'rproxy', aliases: ['web'] }],
+      },
+    ],
+  };
+  it('should get services from the config', () => {
+    const expected = [
+      {
+        stack: 'services',
+        name: 'visualizer',
+        aliases: [],
+      },
+      {
+        stack: 'app',
+        name: 'rproxy',
+        aliases: ['web'],
+      },
+    ];
+    const actual = getServices(config);
+    expect(actual).to.deep.equal(expected);
```

Comment:
Yeah, I agree. I got into doing what your describing when using Ava and it's much better, but didn't really think about it this time and fell back to old habits. :-) Not going to change them all now, but will use the `assert` family in future.